src/spyglass/spikesorting/imported.py

```diff
@@ -29,6 +35,14 @@ def make(self, key):
             else:
                 logger.warn("No units found in NWB file")
 
+        from spyglass.spikesorting.merge import SpikeSortingOutput
```

Comment:
If only for draft, plz ignore.

If planned for inclusion, this will cause errors for new instances. If SpikeSortingOutput is not already declared, importing this table will fail because this line will attempt to declare the table within the transaction, raising [this error](https://github.com/datajoint/datajoint-python/blob/master/datajoint/table.py#L92)

Reply:
Is this also a problem for TrodesPosV1?

Reply:
With our current system of `import` order of operations and the `populate_all` logic, no, not in my pytest container. In principle, yes, the impact would be the same and should be adjusted in both places, moving populate responsibility to the merge table. I think we would have trouble replicating the current fluke that makes the difference